dist/api_data.py

```diff
@@ -121,6 +121,9 @@ def __ge__(self, other):
     Config('role', '', r'''
         whether the stable table in a layered data store should lead or follow''',
         choices=['leader', 'follower'], undoc=True),
+    Config('shutdown_checkpoint', 'false', r'''
+        Whether do checkpoint at shutdown''',
```

Comment:
I think `Whether` should be `whether`.  The lower case applies to all entries in `connection_disaggregated_config_common` if you want to fix that while you're in there.